scripts/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[workspace]
```

Comment:
Because of this "workspace" definition file the target compilation directory is `scripts/target/release`.